pages/common/kate.md

```diff
@@ -0,0 +1,32 @@
+# kate
+
+> KDE Text Editor.
+> More information: <https://kate-editor.org/>.
+
+- Launch Kate and open specific files:
+
+`kate {{path/to/file1}} {{path/to/file2}}`
+
+- Open a remote document in Kate:
```

Comment:
```suggestion
- Open a remote document in Kate (read-only):
```

Reply:
No, is an untitle document in memory (or temp file not sure) but you can edit/save.
Maybe using Samba you can edit but you can't save if you don't have write permissions

Reply:
Hmm, then let's leave it as-is.